public/docs/PAYMENT_METHODS_SETUP.md

```
@@ -0,0 +1,343 @@
+   - Cards: 2-7 business days
+   - UPI: 1-3 business days
+   - ACH: 5-7 business days
+   - PayPal: 1-3 business days
+
+---
+
+## 🧪 Testing Payment Methods
+
+### Test Mode Cards (Worldwide)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)